From 73e16a3dbdf1b0b64ea9d6077166c82a862cf9fa Mon Sep 17 00:00:00 2001
From: Xiaokang Qin <xiaokang.qin@intel.com>
Date: Tue, 14 May 2013 17:18:03 +0800
Subject: [PORT FROM MAIN] Bionic : align socket.h with kernel3.4

BZ: 87037

The socket.h file from the bionic needs to be aligned with the
new kernel 3.4.
This patch is a workaround as we have introduced a 100 gap in the
AF declaration between the last value from the upstream and the new
AF_MHI.

Change-Id: Iaeab82260be6fdbfb8be954248c952fb91fbc77c
Orig-Change-Id: I2c314a4118054f8f0359e2104279caf78be7aeec
Signed-off-by: Frederic Predon <frederic.predon@intel.com>
Reviewed-on: http://android.intel.com:8080/92108
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Ravindran, Arun <arun.ravindran@intel.com>
Reviewed-by: Smith, Geoff <geoff.smith@intel.com>
Reviewed-by: Lebsir, SamiX <samix.lebsir@intel.com>
Tested-by: Lebsir, SamiX <samix.lebsir@intel.com>
---
 libc/include/sys/socket.h |    7 ++++++-
 1 files changed, 6 insertions(+), 1 deletions(-)

diff --git a/libc/include/sys/socket.h b/libc/include/sys/socket.h
index 17ba0a1..244edf8 100644
--- a/libc/include/sys/socket.h
+++ b/libc/include/sys/socket.h
@@ -176,7 +176,10 @@ struct ucred {
 #define AF_IEEE802154 36
 #define AF_CAIF 37
 #define AF_ALG 38
-#define AF_MAX 39
+#define AF_NFC 39
+#define AF_MHI 139
+#define AF_RAW 140
+#define AF_MAX 141
 
 #define PF_UNSPEC AF_UNSPEC
 #define PF_UNIX AF_UNIX
@@ -217,6 +220,8 @@ struct ucred {
 #define PF_IEEE802154 AF_IEEE802154
 #define PF_CAIF AF_CAIF
 #define PF_ALG AF_ALG
+#define PF_MHI AF_MHI
+#define PF_RAW AF_RAW
 #define PF_MAX AF_MAX
 
 #define SOMAXCONN 128
-- 
1.7.4.1

